src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -331,11 +361,10 @@ const SelectPanelFooter = ({...props}) => {
     </Box>
   )
 }
-SelectPanel.Footer = SelectPanelFooter
 
-// @ts-ignore todo
-SelectPanel.SecondaryButton = props => {
-  return <Button {...props} size="small" type="button" block />
+// TODO: is this the right way to add button props?
+const SelectPanelSecondaryButton: React.FC<ButtonProps> = props => {
```

Comment:
Honestly this might be fine here (and is all React.FC is doing anyways) 🤷 

```suggestion
const SelectPanelSecondaryButton = (props: ButtonProps) => {
```

Reply:
Will sync with what we decide for https://github.com/primer/react/pull/3929/files#r1407598117